samples/net/mqtt_publisher/boards/cc3220sf_launchxl.conf

```diff
@@ -0,0 +1,26 @@
+# Networking Config:
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_SOCKETS=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Enable SimpleLink WiFi Driver and Socket Offload
+CONFIG_WIFI=y
+CONFIG_WIFI_SIMPLELINK=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Enable Secure Socket Offload
+CONFIG_TLS_CREDENTIAL_FILENAMES=y
```

Comment:
I don't think comment corresponds to the content here.

Reply:
Thanks I'll update.